lib/screens/onboarding/create_account.dart

```diff
@@ -76,53 +77,98 @@ class _CreateAccountState extends State<CreateAccount> {
   }
 
   String _validateAccountName(String val) {
-    if (val.length != 12)
+    if (val.length != 12) {
       return 'Your account name should have exactly 12 symbols';
-    if (RegExp(r'0|6|7|8|9').allMatches(val).length > 0)
+    } else if (RegExp(r'0|6|7|8|9').allMatches(val).length > 0) {
       return 'Your account name should only contain number 1-5';
-    if (val.toLowerCase() != val)
+    } else if (val.toLowerCase() != val) {
       return "Your account name can't cont'n uppercase letters";
+    } else if (RegExp(r'[a-z]|1|2|3|4|5').allMatches(val).length != 12) {
```

Comment:
looks that this check is not working

![Screenshot_2020-01-27_14-23-28](https://user-images.githubusercontent.com/4569866/73153942-9b5bfa80-4110-11ea-8cee-ba33443639f0.png)

Reply:
done.